From 312abe291550b979d77a21e72f47a281776fccaf Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Fri, 4 Nov 2016 22:58:45 +0300
Subject: [PATCH 19/24] ARM64: dts: r8a7795-salvator-x: Set KOE TX31D200VM0BAA
 1280x480 panel

This sets LVDS LCD parameters for KOE TX31D200VM0BAA 1280x480 panel.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi b/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi
index c7497bb..f308059 100644
--- a/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi
@@ -13,20 +13,20 @@
 	lvds: lvds {
 		compatible = "lvds-connector";
 
-		width-mm = <210>;
-		height-mm = <158>;
+		width-mm = <291>;
+		height-mm = <109>;
 
 		panel-timing {
-			/* 1024x768 @60Hz */
-			clock-frequency = <65000000>;
-			hactive = <1024>;
-			vactive = <768>;
-			hsync-len = <136>;
+			/* 1280x480 @60Hz */
+			clock-frequency = <43200000>;
+			hactive = <1280>;
+			vactive = <480>;
+			hsync-len = <20>;
 			hfront-porch = <20>;
-			hback-porch = <160>;
-			vsync-len = <6>;
-			vfront-porch = <3>;
-			vback-porch = <29>;
+			hback-porch = <40>;
+			vsync-len = <15>;
+			vfront-porch = <15>;
+			vback-porch = <20>;
 		};
 	};
 };
-- 
1.9.3

